opencaesar/oml-vision
# Feat/40-layouts-to-viewpoints

## Checklist before submitting a merge request


## Description of contribution

Allow for users to load webview pages from the `viewpoints` directory instead of the `layouts` directory

closes https://github.com/opencaesar/oml-vision/issues/40
 
## Testing performed




## How to Test Expected functionality changes

OML Model:
1. You can use this OML model for testing https://github.com/pogi7/kepler16b-example
2.  Update the `src/vision/layouts` file to `src/vision/viewpoints`.  
3. Verify that all views open

## Additional context
